xinference/client/restful/restful_client.py

```diff
@@ -799,10 +800,8 @@ def launch_model(
         quantization: Optional[str] = None,
         replica: int = 1,
         n_gpu: Optional[Union[int, str]] = "auto",
+        peft_model_config: Optional[PeftModelConfig] = None,
```

Comment:
The entry of client does not import this new class `PeftModelConfig`. This breaks the usage. Just use dict, then convert to `PeftModelConfig` internally.